tools/eos-content-merge

```diff
@@ -68,7 +70,7 @@ class App(object):
 
     def merge_desktop_file(self, outpath, inplace=False):
         # Find the content desktop file
-        inpath = '/usr/share/eos-shell-content/bundle/desktops/' + \
+        inpath = os.getcwd() + '/bundle/desktops/' + \
```

Comment:
I did run it but I forgot to include the change back (or rather undo the change) in the patch...